index.html

```diff
@@ -0,0 +1,553 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        Web Page
+    </title>
+    <meta charset="utf-8">
+    <meta name='viewport' content="width=device-width" , initial-scale=1'>
+    <link rel="stylesheet" href="webpage.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <!-- Header of company page-->
+    <header class="header-style">
+        <div class="head-infos">
+            <i class="fa fa-phone phone"></i> +123-456-7890
+            <i class="fa fa-envelope"></i>info@gmail.com
+        </div>
+        <div class="head-icons">
+            <div class="head-infos">
+                <i class="fa fa-facebook"></i>
+                <i class="fa fa-twitter"></i>
+                <i class="fa fa-google-plus phone"></i>
+                <i class="fa fa-linkedin"></i>
+            </div>
+        </div>
+    </header>
+<!--===================================================================
+first section includes company name n drop down
+=======================================================================-->
+    <div class="flex-container">
```

Comment:
This should be a nav tag.   I see lots of divs.  Use divs as last resort. Stick to semantic html